docs/src/lib/internals.md

```diff
@@ -15,7 +15,11 @@ gennames
 getmaxwidths
 ourshow
 ourstrwidth
+setmultithreading
+singlethreaded
```

Comment:
I would move these two to manual and add `Experimental` section there.

Reply:
Done. I've named the section "Multithreading configuration (experimental)" as sections are currently organized by theme.